tensorflow/core/util/tensor_bundle/BUILD

```diff
@@ -27,7 +27,7 @@ filegroup(
         "naming.h",
         "tensor_bundle.cc",
         "tensor_bundle.h",
-        "//tensorflow/tsl/util/tensor_bundle:mobile_srcs",
+        "//tensorflow/tsl/util:mobile_srcs_byteswaparray",
```

Comment:
this isn't needed, remove (it will already be included by tsl/util).

Reply:
Removed